benchto-driver/src/test/java/com/teradata/benchto/driver/loader/BenchmarkLoaderTest.java

```diff
@@ -122,6 +128,56 @@ public void shouldLoadSimpleBenchmark()
     }
 
     @Test
+    public void shouldLoadAllBenchmarks()
+            throws IOException
+    {
+        // Given no withActiveBenchmarks(...)
+        // When
+        List<Benchmark> benchmarks = loader.loadBenchmarks("sequenceId");
+        // Then
+        Set<String> loadedNames = benchmarks.stream()
+                .map(Benchmark::getName)
+                .collect(toSet());
+
+        assertThat(loadedNames)
+                .contains("concurrent-benchmark", "simple-benchmark");
+    }
+
+    @Test
+    public void shouldLoadBenchmarksFromFileSystem()
+            throws Exception
+    {
+        String dir = new File(getClass().getResource("/unit-benchmarks/simple-benchmark.yaml").toURI()).getParentFile().getAbsolutePath();
+        checkBenchmarksLoading(dir);
+    }
+
+    @Test
+    public void shouldLoadBenchmarksFromFileSystemRelativePath()
+            throws Exception
+    {
+        Path dir = new File(getClass().getResource("/unit-benchmarks/simple-benchmark.yaml").toURI()).getParentFile().toPath();
+        Path benchmarksDir = Paths.get("").toAbsolutePath().relativize(dir.toAbsolutePath());
+        checkState(!benchmarksDir.isAbsolute());
+        checkBenchmarksLoading(benchmarksDir.toString());
+    }
+
+    private void checkBenchmarksLoading(String benchmarksDir)
+    {
+        withBenchmarksDir(benchmarksDir);
+
+        // When
+        List<Benchmark> benchmarks = loader.loadBenchmarks("sequenceId");
+
+        // Then
```

Comment:
If you're we live in a world where it's obvious what a test tests, then I am sure you're wrong :)
Requiring such comments serves as a reminder that code should be clear. Once it's clear, the comments are no longer needed. Up to you.